packages/dapi/lib/externalApis/drive/ErrorCodes.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  INTERNAL: 1,
+  INVALID_ARGUMENT: 2,
+  NOT_FOUND: 3,
```

Comment:
Unknown error 4 is missed? Why don't to reuse all gRPC error code? They all make sense to me in long run.

Reply:
Fixed